asmcomp/asmgen.ml

```diff
@@ -212,7 +215,7 @@ let flambda_gen_implementation ?toplevel ~source_provenance ~backend ppf
           definition })
       (Symbol.Map.bindings constants)
   in
-  end_gen_implementation ?toplevel ~source_provenance ppf
+  end_gen_implementation ?toplevel ~source_provenance ppf None
```

Comment:
It could be lifted (I hope Pierre will have some time to explain me how to do it), but I don't think it should prevent a merge.